lib/ship_quiet_logistics/commands/processor.rb

```diff
@@ -11,20 +11,28 @@ def initialize(blackboard, queue)
 
         @config = ShipQuietLogistics.configuration
       end
-
       def call
         messages_count.times do
           document = next_document
           next if !document || document.nil?
+          case document_type document.namespace
+            when 'RMADocument'
+              config.process_rma_handler.(document)
```

Comment:
I think we have a misunderstanding if we don't define that attribute for configuration it won't be there when the consumer needs it. Look here: https://github.com/DynamoMTL/ship_quiet_logistics/blob/master/lib/ship_quiet_logistics.rb#L48-L58